src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -4828,7 +4828,7 @@ End Namespace
 
         End Sub
 
-        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/13305")>
+        <Fact>
         Public Sub MultipleDefinitionsOfValueTuple()
```

Comment:
>MultipleDefinitionsOfValueTuple [](start = 19, length = 31)

I would expect to see unit-test(s) that specifically target multi-targeting. It looks like this one hits the scenario by accident and could easily be changed in a way that will stop that from happening.